packages/rollup-plugin-include-css/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "rollup-plugin-include-css",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "exports": {
+    "types": "./dist/index.d.ts",
+    "default": "./dist/index.js"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "clean": "rimraf dist",
+    "type-check": "tsc --noEmit",
+    "watch": "rollup -c -w"
+  },
+  "dependencies": {
+    "postcss": "^8.4.38",
+    "postcss-modules": "^6.0.0"
+  },
+  "devDependencies": {
```

Comment:
Thinking about the maintainability of the plugin, I imagine dependabot config in the root will also push PR against the plugin's dependencies as well?

Reply:
Yeah! Definitely 👍